source/features/table-input.tsx

```diff
@@ -65,7 +65,11 @@ function add(anchor: HTMLElement): void {
 					<TableIcon/>
 				</div>
 			</summary>
-			<details-menu className="select-menu-modal position-absolute left-0 hx_rsm-modal rgh-table-input" role="menu">
+			<details-menu
+				className="select-menu-modal position-absolute left-0 hx_rsm-modal rgh-table-input"
+				role="menu"
+				data-targets="action-bar.items" // Enables automatic hiding when it doesn't fit
```

Comment:
the `<details-menu />` is always below the first button, this makes it impossible to ever open the popup, it also never hides the button itself

Reply:
PR welcome